crates/lunatic-control-axum/src/routes.rs

```diff
@@ -50,6 +51,9 @@ pub async fn register(
             get_module: format!("http://{host}/module/{{id}}"),
             add_module: format!("http://{host}/module"),
             get_nodes: format!("http://{host}/nodes"),
+            get_process: format!("http://{host}/process/{{name}}"),
```

Comment:
By the way, we should check how path parms in axum work regarding URL encoding https://en.wikipedia.org/wiki/URL_encoding and also names cannot contain / etc. hmm

Reply:
I *think* that it can contain `/` as long as it's properly url-encoded.

I'll try and write a test for that.